services/provider/server/server.go

```diff
@@ -915,3 +915,19 @@ func extractMonitorIps(data string) ([]string, error) {
 	slices.Sort(ips)
 	return ips, nil
 }
+
+func (s *OCSProviderServer) OnboardStorageClusterPeer(_ context.Context, _ *pb.OnboardStorageClusterPeerRequest) (*pb.OnboardStorageClusterPeerResponse, error) {
+	return &pb.OnboardStorageClusterPeerResponse{}, nil
+}
+
+func (s *OCSProviderServer) OffboardStorageClusterPeer(_ context.Context, _ *pb.OffboardStorageClusterPeerRequest) (*pb.OffboardStorageClusterPeerResponse, error) {
+	return &pb.OffboardStorageClusterPeerResponse{}, nil
+}
+
+func (s *OCSProviderServer) AcknowledgeOnboardingStorageClusterPeer(_ context.Context, _ *pb.AcknowledgeOnboardingStorageClusterPeerRequest) (*pb.AcknowledgeOnboardingStorageClusterPeerResponse, error) {
+	return &pb.AcknowledgeOnboardingStorageClusterPeerResponse{}, nil
+}
+
+func (s *OCSProviderServer) GetMirroringInfo(_ context.Context, _ *pb.MirroringInfoRequest) (*pb.MirroringInfoResponse, error) {
```

Comment:
Same logic as last comment
```suggestion
func (s *OCSProviderServer) GetPeerInfo(_ context.Context, _ *pb.MirroringInfoRequest) (*pb.MirroringInfoResponse, error) {
```